Eastern-Research-Group/csb-rebate-forms-app
# Feature/update bap query fields

## Related Issues:
* CSBAPP-275

## Main Changes:
Update fields used in BAP queries for 2023 PRF data:
* `School_District_Poverty_Rate__c` changed to `Self_Certification_Category__c`
* `Old_Bus_NCES_District_ID__c` added
* `CSB_Funds_Requested__c` changed to `New_Bus_Infra_Rebate_Requested__c`

## Steps To Test:
1. Ask the BAP to change an existing 2023 FRF submission's status to "Accepted"
2. Create a new 2023 PRF submission.
3. Ensure the injected data for the above fields is correct, when creating a new PRF submission (`_bap_district_selfCertify`, `bus_buses.[index].bus_existingNcesId`, `bus_buses.[index].bus_rebate`)

## TODO:
- [ ] Continue to update queries on the BAP team provides a means to get the data we still need for a new PRF submission.
